docs/pod-identity-mode.md

```diff
@@ -32,9 +32,6 @@ Not all steps need to be followed on the instructions for the aad-pod-identity p
     If not, you can run the following using the Azure cli:
 
     ```bash
-    # Assign Reader Role to new Identity for your keyvault
-    az role assignment create --role Reader --assignee <principalid> --scope /subscriptions/<subscriptionid>/resourcegroups/<resourcegroup>/providers/Microsoft.KeyVault/vaults/<keyvaultname>
```

Comment:
Reader role isn't required because the AKV provider doesn't perform any operation on the vault. Just setting the keyvault access-policies should be sufficient permissions for fetching the contents.